### ISSUE TITLE
Useless const spa?

### ISSUE THREAD

**Issue by frederikhors** [rixo/svelte-template-hot#13]
Now we have this, but where it's used?

```js
const spa = false
```

**Comment by rixo** [comment]
It was used to add the --single flag to sirve, in the custom server plugin that has changed during last merge.

You need that for non hashed-based client side routing. It's an issue that comes quite often with people that want to do SPAs. The point was to help with this situation.

We can probably remove this now. What do you think?

How is the SPA hard point addressed in the official template currently?

**Comment by frederikhors** [comment]
Do you refer to this? https://github.com/sveltejs/template/pull/116

I think we can enlight this in Readme and remove that `const`.

I'm using `--single` in `serve` function.

**Comment by rixo** [comment]
Yes that's what I'm referring to. Apparently unsolved upstream... I guess we can just drop the const for now and follow suit if they take any decision :shrug:
